app/models/search_builder.rb

```diff
@@ -1,7 +1,15 @@
 # frozen_string_literal: true
+
+# Note: In Sufia 7.x at all, this is _not_ used by the CatalogController,
```

Comment:
Still not sure what the implications of this are. Is there a place where the date facet will not be available?